lib/Form/Util/Styles.php

```diff
@@ -8,13 +8,16 @@ class Styles {
   private $defaultStyles = <<<EOL
 /* form */
 .mailpoet_form {
-  padding: 10px;
+  padding: 20px;
 }
 
 /* columns */
 .mailpoet_column_with_background {
   padding: 10px;
 }
+.mailpoet_form_columns.mailpoet_paragraph {
```

Comment:
I can't apply the fixed bar fix as-is, because the class structure of a slide-in form in the Jira task (spec A, URL is in the screenshot) is different: `.mailpoet_form_slide_in` is applied to both `div` and `form`, `.mailpoet_paragraph` is applied to `.mailpoet_form_columns`, there is no `.mailpoet_paragraph` inside `.mailpoet_form_column` or in `p` tags. So I have to modify this a bit. You're welcome to check out.